profile_bluesky/startup/instrument/devices/aps_source.py

```diff
@@ -4,7 +4,7 @@
 """
 
 __all__ = [
-    'aps',
+   # 'aps',
```

Comment:
?

Reply:
While I was at the beamline, for some reason I don't understand, the `apstools.devices.ApsMachineParametersDevice(name="aps")` was failing, so I commented it out. It just started working later and I forgot to uncomment this line. I couldn't reproduce the issue.

Reply:
There was a problem with the access to the aps_cycle name via the Data Management package.  I believe that is now fixed.